app/models/atedModels.scala

```diff
@@ -96,9 +98,21 @@ case class DisposeLiabilityReturn(atedRefNo: String,
                                   timeStamp: DateTime = DateTime.now(DateTimeZone.UTC))
 
 object DisposeLiabilityReturn {
+  implicit val mongoJodaFormats: Format[DateTime] = MongoJodaFormats.dateTimeFormat
+
   def formats(implicit crypto: CompositeSymmetricCrypto): OFormat[DisposeLiabilityReturn] = {
     implicit val bankDetailsModelFormat: Format[BankDetailsModel] = BankDetailsModel.format
 
-    Json.format[DisposeLiabilityReturn]
+    val format: OFormat[DisposeLiabilityReturn] = (
```

Comment:
think you can retain the original line if you import the MongoJodaFormats.Implicits._ at top of module which will make the LocalDate in PendingClient and DisposeLiability read/write correctly.